### DISCUSSION TITLE
Turning off "published a release" notifications

### DISCUSSION THREAD

**Discussion by k0kubun** [community/community#13148] [Other Feature Feedback, Questions, & Ideas]
I like the "For you" feed in general. I see stuff much more relevant to me than the "Following" feed.

This actually applies to both "For you" and "Following", but is it possible to filter out "published a release" notifications?

I get work-related service release notifications there even when I come to GitHub for hobby OSS, and I've never appreciated it. TBH I'm not sure if I care about OSS release notifications either especially when people release a new version for every change and it almost seems like a "commit push" feed.

**Comment by bdresser** [comment]
Hey @k0kubun, thanks for the feedback! We have plans to experiment with some controls that will let folks filter out particular types of feed content — hang tight.

**Reply by k0kubun** [reply]
That's good news. Thank you!

**Comment by jfullstackdev** [comment]
and 

there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by k0kubun** [reply]
Nice! Thank you for sharing this feature.
